Storage/SQL/BrowserTable.swift

```diff
@@ -234,6 +238,12 @@ public class BrowserTable: Table {
             "title TEXT" +
         ") "
 
+    let activityStreamBlocklistCreate =
+        "CREATE TABLE IF NOT EXISTS \(TableActivityStreamBlocklist) (" +
+            "id INTEGER PRIMARY KEY AUTOINCREMENT, " +
+            "url TEXT NOT NULL UNIQUE " +
```

Comment:
I know we're not implementing the expiration logic Android has but having a creation_date column might help us in the future and save a migration. Check out the metadata PR I was working on for some syntax for this.